packages/api/legacy-require-handler.cjs

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable import/no-commonjs */
+
+/**
+ * You can regenerate the type declarations for this file by running the following:
+ * ```sh
+ * npx tsc legacy-require-handler.cjs --checkJs --declaration --emitDeclarationOnly
+ * ```
+ */
+
+/**
+ * In `api` v6, this package was a JS library used JavaScript Proxies to dynamically generate SDKs at runtime.
+ * This functionality has been sunset in `api` v7 to focus on its CLI, which builds strongly-typed SDKs.
+ * This file is here to throw errors for users that attempt to use the legacy library.
+ *
+ *
+ * @param {unknown} input
```

Comment:
```suggestion
 * @param {string} input
```

Reply:
i thought about this but we technically allowed OAS documents back in the day:

https://github.com/readmeio/api/blob/709c54bdb8fd64672f3f06f165410afe7798713d/packages/api/src/index.ts#L201

and I kinda like how there's currently just one TS error rather than another one that's basically a red herring:

![CleanShot 2023-10-27 at 16 00 54@2x](https://github.com/readmeio/api/assets/8854718/73a5f903-292d-4fa8-b1aa-10fbe71d1088)

Reply:
ah true. tbh i think that was only ever exposed for unit tests so i doubt anyone in the wild used it but leaving this as `unknown` is cool.